reproductions/OrleansCrash/App.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
+    <gcServer enabled="true"/>
```

Comment:
Are these required to trigger the exception?

Reply:
They are simply recommended settings from orleans docs.